src/pages/member/UserHome.js

```diff
@@ -1,15 +1,175 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Helmet } from 'react-helmet';
+import Typography from '@material-ui/core/Typography'
+import EventCard from './EventCard'
+import { fetchBackend, updateEvents } from "../../utils";
+import { COLOR } from '../../constants/Constants'
+import House from '../../assets/house.svg'
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import { makeStyles } from '@material-ui/core/styles'
+import { connect } from "react-redux";
+
+const styles = {
+    home: {
+        color: COLOR.TITLE_GREEN,
+        fontStyle: 'normal',
+        fontWeight: 'bold',
+        paddingLeft: '85px',
+        paddingTop: '50px',
+    },
+    page: {
+        height: '100vh'
+    },
+    container: {
+        display: 'flex'
+    },
+    header: {
+        fontStyle: 'normal',
+        fontWeight: 'bold',
+        fontSize: '36px',
+        paddingLeft: '76px',
+        paddingTop: '45px'
+    },
+    reward: {
+        fontStyle: 'normal',
+        fontWeight: 'normal',
+        fontSize: '22px',
+        paddingLeft: '76px',
+        paddingTop: '15px',
+    },
+    house: {
+        position: 'absolute',
+        left: '685px',
+        top: '110px'
+    }
+}
+
+const useStyles = makeStyles({
+    root: {
+        width: '719px',
+        marginTop: '27px'
+    },
+})
+
+function UserHome(props) {
+    const [featuredEvent, setFeaturedEvent] = useState()
+    const [nextEvent, setNextEvent] = useState()
+    const getFeaturedEvent = () => {
+        if (props.events && props.events.length) {
+            setFeaturedEvent(props.events[Math.floor(Math.random() * (props.events.length - 1))]);
+        }
+    }
+
+    /**
+     * gets the next event that the user is registered for
+     * verifies that the event is after the current time
+     * sets next event to 'None Registered!' if no events found
+     */
+    const getNextEvent = async () => {
+        const params = new URLSearchParams({
+            id: props.user.id
+        })
+        await fetchBackend(`/registrations?${params}`, 'GET')
+            .then(async response => {
+                if (response && response.size > 0) {
+                    //iterate over events - the first one that is found in registrations is the closest event assuming that events are already sorted by date
+                    let found = false;
+                    if (props.events) {
+                        props.events.forEach(event => {
```

Comment:
can we just get the first event from the array instead of doing a foreach?

Reply:
i think it depends whether we want to display BizTech's next upcoming event or the closest event thats upcoming for the user that they're registered for. If it's the former, then yes we can do that - but i think the current implementation is needed to account for whether or not they're signed up for the closest event

Reply:
oh right ok